ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -164,8 +165,13 @@ const EditGasItem = ({ priorityLevel }) => {
             <EditGasToolTip
               t={t}
               priorityLevel={priorityLevel}
-              maxFeePerGas={maxFeePerGas}
-              maxPriorityFeePerGas={maxPriorityFeePerGas}
+              maxFeePerGas={maxFeePerGas || maxFeePerGasValue}
```

Comment:
What is the difference between these two values? Is there a better name that we can give that would designate where these come from? I can see this being confusing.